components/style.js

```diff
@@ -23,7 +23,8 @@ export default EStyleSheet.create({
         fontSize: 30,
         lineHeight: 36,
         fontWeight: "bold",
-        fontFamily: "Helvetica Neue, sans-serif"
+        fontFamily: "Helvetica Neue, sans-serif",
```

Comment:
I think in general lets not use this font...

Reply:
@GeorgeBellTMH No problem. I haven't gone through the whole code checking for this font yet but I will.